libass/x86/x86inc.asm

```diff
@@ -35,13 +35,24 @@
 ; to x264-devel@videolan.org .
 
 %ifndef private_prefix
-    %define private_prefix ass
+    %define private_prefix x264
```

Comment:
I imported the latest version outright, and decided to define `private_prefix` in `configure.ac` this time, so we don't have to modify x86inc.asm from "upstream".